CHANGELOG.md

```diff
@@ -0,0 +1,58 @@
+# Page Title Helper Change Log
+
+All notable changes to this project will be documented in this file.
```

Comment:
Reference somewhere to http://keepachangelog.com/en/0.3.0/ as well or is this smth for CONTRIBUTING file?